tests/Common/DevBranchesTest.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+namespace Keboola\Test\Common;
+
+use Keboola\StorageApi\Client;
+use Keboola\StorageApi\ClientException;
+use Keboola\StorageApi\DevBranches;
+use Keboola\StorageApi\Options\TokenCreateOptions;
+use Keboola\Test\Helpers\ClientsProvider;
+use Keboola\Test\StorageApiTestCase;
+
+class DevBranchesTest extends StorageApiTestCase
```

Comment:
Když ty tvoje api cally vrací custom string code `devBranch.XY`. Měl bys to v těch testech také testovat. Aby když to v budoucnu změníme/rozbijeme tak aby to zachytily testy.

Reply:
Fixed